### PR TITLE
BZ1997337: move SAN certificate feature from deprecated to removed features list in 4.6

### PR DESCRIPTION
**Pull request by chinmayi-chandrasekar**
Applies only to 4.6

Docs preview link: https://deploy-preview-36324--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes?utm_source=github&utm_campaign=bot_dp

https://bugzilla.redhat.com/show_bug.cgi?id=1997337

Requires ack from @yaoli-redhat @sttts @Anandnatraj

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -1446,6 +1441,11 @@ All MongoDB-based samples have been replaced, deprecated, or removed.
 ==== External provisioner for AWS EFS (Technology Preview) feature has been removed
 The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview feature has been removed and is no longer supported.
 
+[id="ocp-4-6-tls-common-name"]
+==== TLS verification falling back to the Common Name field
```
Comment (sagidlow) [inline release_notes/ocp-4-6-release-notes.adoc:1445]:
```suggestion
==== TLS verification falling back to the *Common Name* field
```

Per the IBM Style Guide, we should be bolding field names.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -1446,6 +1441,11 @@ All MongoDB-based samples have been replaced, deprecated, or removed.
...
+
+The behavior of falling back to the Common Name field on X.509 certificates as a host name when no Subject Alternative Names are present is removed. Certificates must properly set the Subject Alternative Names field.
```
Comment (sagidlow) [inline release_notes/ocp-4-6-release-notes.adoc:1447]:
```suggestion
The behavior of falling back to the *Common Name* field on X.509 certificates as a host name when no Subject Alternative Names are present is removed. Certificates must properly set the *Subject Alternative Names* field.
```
Per the IBM Style Guide, we should be bolding field names. 

Subject Alternative Names are capitalized like acronyms. If it is not a proper noun then it needs to be in lower case. If it is a field, then it needs to be bolded and identified as a field.

**Review by sagidlow — COMMENTED** [review]
I left two comments, otherwise LGTM.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (chinmayi-chandrasekar) [reply release_notes/ocp-4-6-release-notes.adoc]: Addressed.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (chinmayi-chandrasekar) [reply release_notes/ocp-4-6-release-notes.adoc:1445]: Addressed.

Comment (sherryylch) [inline release_notes/ocp-4-6-release-notes.adoc:1445]: suggest add keyword 'removed', like 'TLS verification falling back to the Common Name field removed'

Comment (yaoli-redhat) [inline release_notes/ocp-4-6-release-notes.adoc:1445]: suggest to add keyword 'removed', like 'TLS verification falling back to the Common Name field removed'

**Comment by vikram-redhat** [comment]
Since the support status of a feature is changing after GA, we need to follow change management for this.

**Pull request merged**